app/models/item.rb

```diff
@@ -9,20 +9,22 @@ class Item < ApplicationRecord
   has_many :invoices, through: :invoice_items
   has_many :transactions, through: :invoices
 
+  scope :find_match_by_name, ->(name) do
```

Comment:
Oh wow! How do scopes work? Would love to talk about these as this solution is so clean

Reply:
Sure!